doc/nrf/ug_matter_gs_testing_thread_separate_otbr_linux_macos.rst

```diff
@@ -42,6 +42,15 @@ Configuring the environment
 
 To configure and use Thread Border Router and CHIP Tool for Linux or macOS on separate devices, complete the following steps.
 
+.. note::
+      This setup is showcased in the "Developing Matter 1.0 products with nRF Connect SDK", embedded below.
+
+      .. raw:: html
+
+         <div style="text-align: center;">
+         <iframe width="560" height="315" src="https://www.youtube.com/embed/9Ar13rMxGIk?t=805" title="Developing Matter 1.0 products with nRF Connect SDK webinar video" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Does this link work? When i manually enter this link it does not jump to the specified time?

Reply:
It does for me.